bigquery/api/Snippets/QueryWithTimestampParameters.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright(c) 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START bigquery_query_params_timestamps]
+
+using Google.Cloud.BigQuery.V2;
+using System;
+
+public class BigQueryQueryWithTimestampParameters
+{
+    public void QueryWithTimestampParameters(
+        string projectId = "your-project-id"
+    )
```

Comment:
nit: same here, parentheses always after the last parameter.

Reply:
Done.